chain/client/src/stateless_validation/validate.rs

```diff
@@ -119,20 +125,47 @@ pub fn validate_chunk_contract_accesses(
     Ok(true)
 }
 
+fn validate_as_chunk_validator(
```

Comment:
nit. the name of this function makes what is being validated unclear. so this is validating that chunk production info is valid and the chunk validator also belongs to that chunk. maybe we can just inline these two calls at the call point? or maybe rename to `validate_chunk_relevant_as_validator` or something like that?

Reply:
renamed to `validate_chunk_relevant_as_validator`